packages/core/src/Modules/Cashier/Containers/on-ramp.jsx

```diff
@@ -10,20 +10,52 @@ import 'Sass/app/modules/on-ramp.scss';
 const OnRamp = ({
     is_onramp_modal_open,
     filtered_onramp_providers,
+    menu_options,
     onramp_popup_modal_title,
     setIsOnRampModalOpen,
     should_show_dialog,
     resetPopup,
     onMount,
     onUnmount,
+    history,
 }) => {
+    const [selected_cashier_path, updateSelectedChaierPath] = React.useState('/cashier/on-ramp');
+    React.useEffect(() => {
+        if (menu_options?.length) {
```

Comment:
There is no need to use optional chaining for `menu_options`, it is always an array. Please also don't forget to define PropTypes for each new prop you're adding.